### PR TITLE
Show release version in the about/cohort tab

### PR DESCRIPTION
**Pull request by creilly8**
## Description
Closes issue #2227.

Show the release version at the bottom of the abort/cohort tab as a link to github container page, same as the app header. Test with http://localhost:3000/?noheader=1&massnative=hg38-test,TermdbTest. 

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/mass/nav.js [hunk 1]
```diff
@@ -624,6 +626,20 @@ function setRenderers(self) {
 				.text(selectCohort.asterisk)
 		}
 	}
+
+	self.showReleaseVersion = appState => {
+		if ((!appState?.termdbConfig?.selectCohort && !appState.termdbConfig?.massNav?.tabs?.about) || self.pkgver == null)
```
Comment (siosonel) [inline client/mass/nav.js:631]: A subjective preference, but I'd prefer showing the version somewhere in the top right corner of the subheader. Or, maybe have a `Version` button/text or info icon next to the session button, that can be clicked to show a menu with version info.

Comment (xzhou82) [reply client/mass/nav.js:631]: for a ds without subcohort, it should always have the About tab.

Comment (creilly8) [reply client/mass/nav.js:631]: Ok, do you want that complete in this PR or in a separate one? Also if there's not other information besides the release version, should the about tab be the active tab or should the charts tab show by default?

Comment (xzhou82) [reply client/mass/nav.js:631]: 1. no worry. add persistent About tab in later PR.
2. if no other info, the About tab just show dslabel and release version
3. About tab should be the default active tab

**Review by xzhou82 — APPROVED** [review]

**Comment by xzhou82** [comment]
please standardize its display here and app header. use `Release version: hyperlinked text`

**Pull request merged**